docs/library/pyb.CAN.rst

```diff
@@ -67,6 +67,10 @@ Methods
      - *auto_restart* sets whether the controller will automatically try and restart
        communications after entering the bus-off state; if this is disabled then
        :meth:`~CAN.restart()` can be used to leave the bus-off state
+     - *baudrate* if a baudrate other than 0 is provided, this function will automatically try to
+       calculate a CAN bit-timing that satisfies both the baudrate and the desired *sample_point*.
```

Comment:
maybe be clear that prescaler/bs1/bs2 are overridden, eg `... try to calculate a CAN bit-timing (overriding *prescaler*, *bs1* and *bs2*) that satisfies ...`

Reply:
done.